pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -161,6 +162,32 @@ func (h *upgradeHandler) OnChanged(key string, upgrade *harvesterv1.Upgrade) (*h
 			return h.upgradeClient.Update(toUpdate)
 		}
 
+		logrus.Info("Check minimum upgradable version")
+		minUpgradableVersion, err := repo.getMinUpgradableVersionStr()
```

Comment:
Maybe we can refactor the code `getInfoStr()` to `getInfo()` to avoid creating a new method?

Reply:
Since both `getInfoStr()` and `getMinUpgradableVersionStr()` are one-off methods, maybe we can just simply extract what we need from `getInfo()` in the caller function. Is that what you meant?

Reply:
Updated it, thanks.